easybuild/easyconfigs/i/Infernal/Infernal-1.1rc1-ictce-5.3.0.eb

```diff
@@ -15,15 +15,15 @@ easyblock = 'ConfigureMake'
 name = 'Infernal'
 version = '1.1rc1'
 
-homepage = 'http://infernal.janelia.org/'
+homepage = 'http://eddylab.org/infernal/'
 description = """Infernal ('INFERence of RNA ALignment') is for searching DNA sequence databases
  for RNA structure and sequence similarities."""
 
 toolchain = {'name': 'ictce', 'version': '5.3.0'}
 toolchainopts = {'optarch': True, 'pic': True}
 
 sources = [SOURCELOWER_TAR_GZ]
-source_urls = ['ftp://selab.janelia.org/pub/software/%s' % name.lower()]
+source_urls = ['http://eddylab.org/infernal/%s' % name.lower()]
```

Comment:
I mean, the tarball for `1.1rc` is not available in the new location...

I'd suggest to just remove `source_urls` instead then, and replace it with a comment, like so:

```python
# used to be at ftp://selab.janelia.org/pub/software/infernal, but not anymore (nor somwhere else)
sources = [SOURCELOWER_TAR_GZ]